move/operators/sources/operators/operators.move

```diff
@@ -0,0 +1,319 @@
+module operators::operators {
+    use sui::bag::{Self, Bag};
+
+    // -----
+    // Types
+    // -----
+
+    /// The `OwnerCap` capability representing the owner of the contract.
+    public struct OwnerCap has key, store {
+        id: UID,
+    }
+
+    /// The `OperatorCap` capability representing an approved operator.
+    public struct OperatorCap has key, store {
+        id: UID,
+    }
+
+    /// The main `Operators` struct storing the capabilities and operator IDs.
+    public struct Operators has key {
+        id: UID,
+        operator_ids: vector<ID>,
+        caps: Bag,
+    }
+
+    // -----
+    // Setup
+    // -----
+
+    /// Initializes the contract and transfers the `OwnerCap` to the sender.
+    fun init(ctx: &mut TxContext) {
+        transfer::share_object(Operators {
+            id: object::new(ctx),
+            operator_ids: vector::empty(),
+            caps: bag::new(ctx),
+        });
+
+        transfer::public_transfer(OwnerCap {
+            id: object::new(ctx),
+        }, tx_context::sender(ctx));
```

Comment:
```suggestion
        let cap = OwnerCap {
            id: object::new(ctx),
        };

        transfer::transfer(cap, ctx.sender());
```
use move 2024 syntax. And refer to the pattern used in gateway contract, public_* isn't needed since object is defined in this module